crates/reporting/src/error/type.rs

```diff
@@ -1655,6 +1655,12 @@ fn format_category<'b>(
             alloc.concat([this_is, alloc.text(" a Unicode scalar value")]),
             alloc.text(" of type:"),
         ),
+        IngestedFile => (
+            // TODO: is this what we actually want for the error message here.
```

Comment:
Ditto here on not really being sure what we want to print.

Reply:
+1 on getting the file name in here. If this becomes an error that is generated during solving as I described above, you will get the actual type of the provided ingested file.